Escape commas in values

Rushi98 opened the pull request:
Enclose all values in double quotes so that excel can recognise commas present in
values from the field separator comma.

gaurabdg commented:
I've cleaned up the repo by removing the ide config files, so that we dont face unnecessary conflicts. 
Can you please update your fork and submit a fresh PR?

Rushi98 committed:
Escape commas in values

Enclose all values in double quotes so that excel can recognise commas present in
values from the field separator comma.

Rushi98 commented:
[WIP]

Tracked in #3
